src/components/Organisms/Footer/Footer.js

```diff
@@ -15,14 +15,16 @@ import {
 } from './Footer.style';
 
 const Footer = ({
-  navItems, footerCopy, campaign, ...rest
+  navItems, footerCopy, campaign, additionalLegalLine, ...rest
 }) => {
   // Remove white space between words
   const campaignName = campaign.replace(/\s/g, '').toLowerCase();
+
   return (
     <div>
       <FooterWrapper navItems {...rest}>
         <InnerWrapper>
+          <Text style={{ textAlign: 'left' }} align="left" tag="p" color="white">{additionalLegalLine}</Text>
```

Comment:
Might be worth making this conditional, as currently the tag is showing appearing in the DOM but with no content:

```jsx
{additionalLegalLine && <Text style={{ textAlign: 'left' }} align="left" tag="p" color="white">{additionalLegalLine}</Text>}
          
```

```html
<p color="white" class="sc-gsTEea jZslOC"></p>
```